src/ApparentHorizons/StrahlkorperGr.hpp

```diff
@@ -468,4 +468,14 @@ double irreducible_mass(double area);
  */
 double christodoulou_mass(double dimensionful_spin_magnitude,
                           double irreducible_mass);
+
+/*!
+ * Compute the pointwise radial distance between two Strahlkorpers with the same
```

Comment:
Oops, didn't see this before. Add `\ingroup SurfacesGroup` to be consistent with the rest of the functions.